tests/e2e/e2e_exec_test.go

```diff
@@ -284,6 +284,44 @@ func (s *IntegrationTestSuite) execBankSend(
 	s.executeGaiaTxCommand(ctx, c, gaiaCommand, valIdx, s.expectErrExecValidation(c, valIdx, expectErr))
 }
 
+func (s *IntegrationTestSuite) execBankMultiSend(
+	c *chain,
+	valIdx int,
+	from string,
+	to []string,
+	amt string,
+	fees string,
+	expectErr bool,
+	opt ...flagOption,
+) {
+	// TODO remove the hardcode opt after refactor, all methods should accept custom flags
```

Comment:
TODO copied from [ExecBankSend](https://github.com/cosmos/gaia/blob/57fb79e0337fbb93eb0dcc80f111b190d223168d/tests/e2e/e2e_exec_test.go#L260) and will be addressed in another refactoring.